schemasheets/template_wizard.py

```diff
@@ -0,0 +1,292 @@
+import csv
+import logging
+import os
+import pprint
+from typing import List, Dict
+
+import click
+import click_log
+import yaml
+from linkml_runtime import SchemaView
+from linkml_runtime.dumpers import yaml_dumper
+
+from schemasheets.schema_exporter import SchemaExporter
+
+logger = logging.getLogger(__name__)
+click_log.basic_config(logger)
+
+# todo: reflect the classes that use a slot on slot-only rows
+
+# todo: work on YAML-serialized columns like annotation and alt_descriptions
+# todo: style guide, esp use of external terms and consistent grammar
+
+# todo add customized first header BEFORE the unfiltered sheet is written
+
+# todo: group and or describe mappings
+# todo: comments, notes, todos distinctions
+
+# todo: generate third row for internal separators and annotation names
+
```

Comment:
use python dataclasses instead of init methods

Reply:
I think you can resolve this comment, looks lovely with dataclasses!